java-client/session/src/main/java/io/deephaven/client/impl/TableObject.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.client.impl;
+
+import io.deephaven.client.impl.TableHandle.TableHandleException;
+import io.deephaven.qst.table.TableSpec;
+import io.deephaven.qst.table.TicketTable;
+
+/**
+ * A {@value TYPE} object.
+ */
+public final class TableObject extends ServerObjectBase implements ServerObject {
+
+    public static final String TYPE = "Table";
+
+    TableObject(Session session, ExportId exportId) {
+        super(session, exportId);
+        checkType(TYPE, exportId);
+    }
+
+    /**
+     * Creates a table handle.
+     *
+     * <p>
+     * Note: the table handle lifecycle is managed separately from {@code this}; the caller is still responsible for
```

Comment:
Why is that true? Won't either one release the ticket that references the table?

Reply:
`io.deephaven.client.impl.TableHandleManager#execute(io.deephaven.qst.table.TableSpec)` under the covers does batch-able ops (TableSpec); in this respect, it does a `FetchTable`, and isn't even wired up for `GetExportedTableCreationResponse`.

It's possible that we could extend the implementation to support creating `TableHandle` via exports you already have, but that would take some amount of extra work.

Reply:
Ok, so fixing this oversight will be a breaking change later, since clients today may close the TableObject after the TableHandle is a real Table, but after the fix closing the TableObject will shut down the TableHandle? Seems like it might be worth it to fix now, rather than break later?

Reply:
I don't expect we'd break this in the future; we can just add a new method.